ansible/inventory/multi_inventory.py

```diff
@@ -5,29 +5,137 @@
 '''
 # vim: expandtab:tabstop=4:shiftwidth=4
 
+from string import Template
 from time import time
 import argparse
-import yaml
-import os
-import subprocess
-import json
+import atexit
+import copy
 import errno
 import fcntl
-import tempfile
-import copy
-from string import Template
+import json
+import os
 import shutil
+import subprocess
+import sys
+import tempfile
+import yaml
 
 CONFIG_FILE_NAME = 'multi_inventory.yaml'
 DEFAULT_CACHE_PATH = os.path.expanduser('~/.ansible/tmp/multi_inventory.cache')
+FILE_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)))
+
+class MultiInventoryAccount(object):
+    """Object to represent an account"""
+    def __init__(self, name, config, cache_path, cache_age=1800):
+        self.name = name
+        self.config = config
+        self._account_data = None
+
+        # set to global cache_age
+        self.cache_age = cache_age
+        # if an account setting exists, set to account cache_age
+        if self.config.has_key('cache_age'):
+            self.cache_age = self.config['cache_age']
+
+        # set to global account cache_path
+        self.cache_path = cache_path
+        # if an account setting exists, set to account cache_path
+        if self.config.has_key('cache_path'):
+            self.cache_path = self.config['cache_path']
+
+        # collect a list of clusters for this account
+        self.clusters = []
+        if self.config.has_key('cluster_vars') and \
+            self.config['cluster_vars'].has_key('clusters'):
+            self.clusters = self.config['cluster_vars']['clusters'].keys()
+
+
+    @property
+    def account_data(self):
```

Comment:
`account_data` feels like it's the config data.

Consider renaming this to `account_inventory`.